install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[openshift-prometheus-web]
+==== Accessing the Prometheus Web UI
+
+The Prometheus server automatically exposes a Web UI at `localhost:9090`. You
+can access the Prometheus Web UI with the `view` role.
+
+[openshift-prometheus-config]
+==== Configuring Prometheus for OpenShift
+
+Example Prometheus rules file:
+----
+# additional prometheus rules file
+openshift_prometheus_additional_rules_file: null
+
+# All the required exports
+openshift_prometheus_pv_exports:
+-  prometheus
+-  prometheus-alertmanager
+-  prometheus-alertbuffer
+# PV template files and their created object names
+  openshift_prometheus_pv_data:
+  -  pv_name: prometheus
+     pv_template: prom-pv-server.yml
+     pv_label: Prometheus Server PV
+  -  pv_name: prometheus-alertmanager
+     pv_template: prom-pv-alertmanager.yml
+     pv_label: Prometheus Alertmanager PV
+  -  pv_name: prometheus-alertbuffer
+     pv_template: prom-pv-alertbuffer.yml
+     pv_label: Prometheus Alert Buffer PV
+
+# Hostname/IP of the NFS server. Currently defaults to first master
+openshift_prometheus_nfs_server: "{{ groups.nfs.0 }}"
+
+# storage
+openshift_prometheus_storage_type: pvc
+openshift_prometheus_pvc_name: prometheus
+openshift_prometheus_pvc_size: 10G
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_pvc_pv_selector: {}
+
+openshift_prometheus_alertmanager_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: prometheus-alertmanager
+openshift_prometheus_alertmanager_pvc_size: 10G
+openshift_prometheus_alertmanager_pvc_access_modes: [ReadWriteOnce]
+openshift_prometheus_alertmanager_pvc_pv_selector: {}
+openshift_prometheus_cpu_limit: null
+openshift_prometheus_memory_limit: null
+openshift_prometheus_cpu_requests: null
+openshift_prometheus_memory_requests: null
+openshift_prometheus_alertmanager_cpu_limit: null
+openshift_prometheus_alertmanager_memory_limit: null
+openshift_prometheus_alertmanager_cpu_requests: null
+openshift_prometheus_alertmanager_memory_requests: null
+openshift_prometheus_alertbuffer_cpu_limit: null
+openshift_prometheus_alertbuffer_memory_limit: null
+openshift_prometheus_alertbuffer_cpu_requests: null
+openshift_prometheus_alertbuffer_memory_requests: null
+openshift_prometheus_oauth_proxy_cpu_limit: null
+openshift_prometheus_oauth_proxy_memory_limit: null
+openshift_prometheus_oauth_proxy_cpu_requests: null
+openshift_prometheus_oauth_proxy_memory_requests: null
+----
+
+*Prometheus Storage Related Variables*
+
+With each Prometheus component--including Prometheus,
+Alertmanager, alert-buffer, and oauth-proxy--you can set the PV claim by setting
+corresponding role variable, for example:
```

Comment:
Do we need to have a storage sizing guide, like we do for logging/metrics that outlines suggested sizes for the volumes?

Reply:
We should include formulas (using our experience with online), on how fast we fill up the volumes, and how fast data rolls off the volumes, based on the alerts were shipping by default.

Reply:
I agree. I talked these over with Ali, and this will need to come post-3.7 since this is Tech Preview. Would you be willing to file a BZ for these? Thank you!